### PR TITLE
v11 color token updates for search component

### PR DESCRIPTION
**Pull request by kingtraceyj**
This PR updates the color tokens on the Style tab for the search component.

### CODE REVIEW

**Comment by kingtraceyj** [comment]
@aagonzales ack! forgot to update the:

Inputs come in two different colors. The default input color is `$field-01` and is used on `$ui-background` and `$ui-02` page backgrounds. The `--light` version input color is `$field-02` and is used on `$ui-01` page backgrounds.

do we want to explain inline theming here instead?

File: src/pages/components/search/style.mdx [hunk 1]
```diff
@@ -14,12 +14,12 @@ input color is `$field-02` and is used on `$ui-01` page backgrounds.
...
 | Class                            | Property         | Color token |
 | -------------------------------- | ---------------- | ----------- |
-| `.bx--search-input`              | background-color | `$field-01` |
-| `.bx--search--light`             | background-color | `$field-02` |
-| `.bx--search-input`              | border-bottom    | `$ui-04`    |
-| `.bx--search-input`              | text color       | `$text-01`  |
-| `.bx--search-input::placeholder` | text color       | `$text-03`  |
-| `.bx--search-magnifier`          | fill             | `$icon-02`  |
+| `.bx--search-input`              | background-color | `$field` |
+| `.bx--search--light`             | background-color | `$background: inline theme` |
```
Comment (aagonzales) [inline src/pages/components/search/style.mdx:18]: just delete this one. The `light` variant wont actually exist anymore.

File: src/pages/components/search/style.mdx [hunk 2]
```diff
@@ -34,9 +34,9 @@ input color is `$field-02` and is used on `$ui-01` page backgrounds.
 | Class                            | Property         | Color token    |
 | -------------------------------- | ---------------- | -------------- |
 | `.bx--search-input:focus`        | border           | `$focus`       |
-| `.bx--search-input:disabled`     | background-color | `$disabled-01` |
-| `.bx--search-input:disabled`     | text color       | `$disabled-02` |
-| `.bx--search-magnifier:disabled` | fill             | `$disabled-02` |
+| `.bx--search-input:disabled`     | background-color | `$field` |
```
Comment (aagonzales) [inline src/pages/components/search/style.mdx:37]: This would be `$field-disabled`

**Review by joshblack — APPROVED** [review]

**Review by andreancardona — APPROVED** [review]

**Review by aagonzales — APPROVED** [review]
Looks good!

**Pull request merged**